### PR TITLE
Fix Time Input Border & Bidirectionality

### LINKED ISSUE #527: Time input colon underlined on safari
# Issue Description

Time input colon underlined on safari

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [x] Bug
- [ ] Other

## Expected Behavior

Chrome:
screen shot 2017-06-28 at 10 20 59 am

## Current Behavior

Safari:
screen shot 2017-06-28 at 10 20 53 am

Underline under the colon

### LINKED ISSUE #638: Time input bidirectionality
# Issue Description

The time input component is not bidirectional.

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [x] Bug
- [ ] Other

## Expected Behavior

Time input is bidirectional.

## Current Behavior

time-input

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
This PR:

- closes #527. 

- closes #638

- Adds font-size such thats the colon will scale. 

@cerner/terra

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
Added bidi implementation to this PR. Resolves https://github.com/cerner/terra-core/issues/638

File: packages/terra-alert/themeable-variables.json [hunk 1]
```diff
@@ -0,0 +1 @@
...rra-table-tfoot-tr-border-bottom":"1px solid #c8cacb","--terra-table-tfoot-tr-border-top":"0","--terra-table-subheader-background-color":"#c8cacb","--terra-table-subheader-color":"#1c1f21","--terra-table-subheader-font-size":"1rem","--terra-table-subheader-font-weight":"bold","--terra-table-sort-indicator-color":"#1c1f21","--terra-table-cell-min-width-tiny":"3rem","--terra-table-cell-min-width-small":"4.25rem","--terra-table-cell-min-width-medium":"5.75rem","--terra-table-cell-min-width-large":"8.5rem","--terra-table-cell-min-width-huge":"10.5rem","--terra-table-cell-padding-bottom":"0.5em","--terra-table-cell-padding-left":"0.75em","--terra-table-cell-padding-right":"0.75em","--terra-table-cell-padding-top":"0.5em","--terra-table-row-striped-background-color":"#f4f4f4","--terra-table-row-striped-selected-background-color":"#d4ecfb","--terra-table-row-striped-hover-background-color":"#e2f2fc","--terra-table-row-striped-selected-hover-background-color":"#bde1f8","--terra-table-row-selected-background-color":"#ebf6fd","--terra-table-row-selected-border-color":"#0092e0","--terra-table-row-hover-background-color":"#ebf6fd","--terra-table-row-selected-hover-background-color":"#d4ecfb"}}
```
Comment (StephenEsser) [inline packages/terra-alert/themeable-variables.json]: Was this addition intentional?

Comment (emilyrohrbough) [reply packages/terra-alert/themeable-variables.json]: No.

File: packages/terra-time-input/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ Unreleased
 ----------
 ### Fixed
 * Fixed focus management issue between hour and time input in Firefox.
+* Fixed border styling
+* Fix input bidirectionality
```
Comment (StephenEsser) [inline packages/terra-time-input/CHANGELOG.md]: nit: Should the bottom read `Fixed input bidirectionality` ?

**Comment by jordanlawrence15** [comment]
design looks good

**Comment by mmalaker** [comment]
Functional verification completed. The time input fields display without the colon underline and represent time correctly now in RTL. Minor issue with the label and input value above the example will be addressed in #789.
